manifests/n/NoMoreFood/PuTTY-CAC/0.77.0.0/NoMoreFood.PuTTY-CAC.installer.yaml

```
@@ -0,0 +1,30 @@
+PackageIdentifier: NoMoreFood.PuTTY-CAC
+PackageVersion: 0.77.0.0
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+FileExtensions:
+- PPK
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: msi
+  Scope: machine
+  InstallerUrl: https://github.com/NoMoreFood/putty-cac/raw/0.77/binaries/puttycac-64bit-0.77-installer.msi
+  InstallerSha256: 0F19075A9B2F2021921D339C6EF864AA2AF51AF4378D23307E280DEB8B89135D
+  UpgradeBehavior: install
+  ProductCode: '{C8684056-D12C-4D7C-A1B2-8A632885C851}'
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: msi
+  Scope: machine
+  InstallerUrl: https://github.com/NoMoreFood/putty-cac/raw/0.77/binaries/puttycac-0.77-installer.msi
+  InstallerSha256: E334061768E4D6C425AB9E25EF0862B317E4B88E5C3936AA627C810C83BE17C1
+  UpgradeBehavior: install
+  ProductCode: '{8976E767-E217-4D8F-9F25-B882A99A34A2}'
```

```suggestion
  ProductCode: '{BCC3136A-77A0-447C-A7C4-475170C92CE9}'
```
winget-create will automatically do this for you. You can take a look here: https://github.com/microsoft/winget-create

Each MSI has a unique GUID attached to it.